### PR TITLE
Bryce/update aws onboarding

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do?
Update AWS onboarding guides to reflect the new integration tile.

### Motivation
DOCS-3896

### Preview
CloudFormation:
https://docs-staging.datadoghq.com/bryce/update-aws-onboarding/getting_started/integrations/aws/
Manual:
https://docs-staging.datadoghq.com/bryce/update-aws-onboarding/integrations/guide/aws-manual-setup/?tab=roledelegation

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/getting_started/integrations/aws.md
Comment (warrierr) [inline content/en/getting_started/integrations/aws.md:109]:
```suggestion
| **Log Collection:**      | Configure Log Autosubscription for AWS services where there are many resources with logs in different CloudWatch log groups or S3 buckets.
```

File: content/en/getting_started/integrations/aws.md
Comment (warrierr) [inline content/en/getting_started/integrations/aws.md:111]:
```suggestion
| **X-Ray Traces:**        | Supplement serverless traces with [AWS X-Ray][61] data for some AWS services. 
```

File: content/en/integrations/guide/aws-manual-setup.md
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:58]:
```suggestion
1. In the [AWS Integration configuration page][6], click **Add AWS Account**, and then select **Manually**.
```
Should be new link: https://app.datadoghq.com/integrations/amazon-web-services

File: content/en/getting_started/integrations/aws.md
Comment (warrierr) [inline content/en/getting_started/integrations/aws.md:80]:
```suggestion
3. Configure the integration's settings under the **Automatically using CloudFormation** option.  
```

**Review by warrierr — CHANGES_REQUESTED** [review]
@buraizu  Added comments/suggestions

File: content/en/getting_started/integrations/aws.md
Comment (warrierr) [inline content/en/getting_started/integrations/aws.md:28]:
```suggestion
      text: 'Monitor your Graviton3-powered EC2 instances with Datadog'
```

File: content/en/getting_started/integrations/aws.md
Comment (buraizu) [reply content/en/getting_started/integrations/aws.md]: Thanks, added back in per our discussion

File: content/en/getting_started/integrations/aws.md
Comment (buraizu) [reply content/en/getting_started/integrations/aws.md]: Thanks for pointing this out, I think I got everything, but let me know if you notice anything else

File: content/en/getting_started/integrations/aws.md
Comment (buraizu) [reply content/en/getting_started/integrations/aws.md]: Got it, updated!

File: content/en/getting_started/integrations/aws.md
Comment (buraizu) [reply content/en/getting_started/integrations/aws.md]: Thanks for confirming, removed it from here, but I'll open a separate PR in Dogweb to add it to the main docs

File: content/en/integrations/guide/aws-manual-setup.md
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md]: Thanks, updated!

File: content/en/integrations/guide/aws-manual-setup.md
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md]: Got it, removed

File: content/en/getting_started/integrations/aws.md
Comment (buraizu) [reply content/en/getting_started/integrations/aws.md]: Thanks for catching that!

**Comment by buraizu** [comment]
Note: Labeling with `Do Not Merge` until release is approved by PM

File: content/en/getting_started/integrations/aws.md
Comment (kayayarai) [inline content/en/getting_started/integrations/aws.md:86]:
```suggestion
    **c.** Optionally, send logs and other data to Datadog with the [Datadog Forwarder Lambda][1].  
```

File: content/en/integrations/guide/aws-manual-setup.md
Comment (kayayarai) [inline content/en/integrations/guide/aws-manual-setup.md:93]: Why not use a regular markdown link here? (and a few other places in this page)

File: content/en/integrations/guide/aws-manual-setup.md
Comment (kayayarai) [inline content/en/integrations/guide/aws-manual-setup.md:120]: use regular markdown link?

File: content/en/getting_started/integrations/aws.md
Comment (kayayarai) [inline content/en/getting_started/integrations/aws.md:84]: My opinion: Don't bold the a., b., c., d. The indenting is enough to set it apart visually.

File: content/en/getting_started/integrations/aws.md
Comment (kayayarai) [inline content/en/getting_started/integrations/aws.md:89]:
```suggestion
5. Click **Launch CloudFormation Template**. This opens the AWS Console and loads the CloudFormation stack. All the parameters are filled in based on your selections in the prior Datadog form, so you do not need to edit those unless desired.  
```

File: content/en/getting_started/integrations/aws.md
Comment (kayayarai) [inline content/en/getting_started/integrations/aws.md:94]:
```suggestion
7. After the stack is created, go back to the AWS integration tile in Datadog and click **Ready!**.
```

File: content/en/getting_started/integrations/aws.md
Comment (kayayarai) [inline content/en/getting_started/integrations/aws.md:92]:
```suggestion
6. Check the required boxes from AWS and click **Create stack**. This launches the creation process for the Datadog stack along with three nested stacks. This could take several minutes. Ensure that the stack is successfully created before proceeding.
```

File: content/en/integrations/guide/aws-manual-setup.md
Comment (kayayarai) [inline content/en/integrations/guide/aws-manual-setup.md:91]:
```suggestion
    **Note:** The role name you enter in the integration tile is case sensitive and must exactly match the role name in AWS.
```

File: content/en/integrations/guide/aws-manual-setup.md
Comment (kayayarai) [inline content/en/integrations/guide/aws-manual-setup.md:60]:
```suggestion
   **Note: Do not close the integration tile or the Datadog site**, as this causes the external ID value to reset.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Some of my comments are more suggestions (the bold on sub-lists, the html links) -- maybe you had good reason to do them they way you did, but they are kind of unusual. so I raised the questions.

File: content/en/getting_started/integrations/aws.md
Comment (buraizu) [reply content/en/getting_started/integrations/aws.md]: Thanks for confirming! I wasn't sure about that myself. Updated.

File: content/en/getting_started/integrations/aws.md
Comment (buraizu) [reply content/en/getting_started/integrations/aws.md]: Thanks for pointing this out, I agree that's better. With that said, I left off the period because I think it might look strange after the exclamation.

File: content/en/integrations/guide/aws-manual-setup.md
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md:60]: Thanks for pointing this out -- just to be clear, is this meant to start the note on the next line? That was my initial intention, but I found that adding just this space didn't accomplish that. The updated branch should show the desired effect, so let me know if you think it looks alright!

File: content/en/integrations/guide/aws-manual-setup.md
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md]: Thanks for pointing this out as well, updated so that the note starts on a new line.

File: content/en/integrations/guide/aws-manual-setup.md
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md:93]: The HTML links on this page were a direct request from the PM, so that they would open new tabs by default. We had some internal discussion around this at the time, but couldn't come up with a way to have markdown links open in a new tab by default (without altering this behavior for the entire site). Let me know if you have any suggestions for that, otherwise I'll leave them as HTML for now.

File: content/en/integrations/guide/aws-manual-setup.md
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md:120]: As above, this is a workaround to have the link open in a new tab.

**Comment by buraizu** [comment]
@kayayarai thanks for your review, I've applied most of your suggestions and gave some additional context on others.

**Review by kayayarai — APPROVED** [review]
One small duplicated word nit, otherwise 👍

File: content/en/integrations/guide/aws-manual-setup.md
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:88]:
```suggestion
18. Return to the AWS Integration configuration page for manually adding an account in Datadog that you had open in another tab, and click the checkbox to confirm the Datadog IAM role was added to the AWS account.
```

File: content/en/integrations/guide/aws-manual-setup.md
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:113]: ^ This is for the line above. This shouldn't be there for access key govcloud/china setup since there is no external ID:

`For more details, see the [How to use an external ID when granting access to your AWS resources to a third party][1] AWS documentation.`

**Review by warrierr — CHANGES_REQUESTED** [review]
Just 2 final comments

**Comment by buraizu** [comment]
@warrierr thanks for pointing out those issues, updated.

**Review by warrierr — APPROVED** [review]

**Pull request merged**
